routes/api/v1/meals.js

```diff
@@ -19,4 +18,42 @@ router.get('/', (request, response) => {
   })
 })
 
+/* POST adds a food to a meal */
+router.post('/:meal_id/foods/:id', (request, response) => {
+  return Meal.findOne({
+    include: Food,
+    where: {
+      id: request.params["meal_id"]
+    }
+  })
+  .then(meal => {
+    return Food.findOne({
+      where: {
+        id: request.params["id"]
+      }
+    })
+    .then(food => {
+      return meal.hasFood(food).then(result => {
+        if (result == true) {
+          response.setHeader('Content-Type', 'application/json');
+          response.status(500).send({ error: 'Meal already has that food' });
+        } else {
+          meal.addFood(food)
+          response.setHeader('Content-Type', 'application/json');
```

Comment:
If meal found and food found, and food not already added to meal, adds the food to the meal.